Nextras/YoutubeApi/Reader.php

```diff
@@ -84,7 +92,7 @@ protected function parseData($data, $videoId)
 	{
 		$data = Nette\Utils\Json::decode($data);
 		if (!isset($data->items[0]->snippet) || !isset($data->items[0]->contentDetails)) {
-			throw new \RuntimeException();
+			throw new \RuntimeException("Not found YouTube response, probably wrong videoId: '{$videoId}'?");
```

Comment:
Empty YouTube response, probably wrong $videoId video id. 
Without the question mark.